ros/src/common/cmake/autoware_build_flags/cmake/autoware_build_flags-extras.cmake

```diff
@@ -9,4 +9,23 @@ if(${CMAKE_VERSION} VERSION_LESS "3.1.0")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 else()
   set(CMAKE_CXX_STANDARD 11)
-endif()
\ No newline at end of file
+endif()
+
+find_package(CUDA QUIET)
+find_package(Eigen3 QUIET)
+
+option(CPU_ONLY "CPU_Only" TRUE)
+
+####### Melodic Checks
+if (NOT CPU_ONLY AND CUDA_FOUND AND "$ENV{ROS_DISTRO}" STREQUAL "melodic" )
```

Comment:
Where else is `CPU_ONLY` used? The logic would read better if it were `ENABLE_CUDA` (i.e. reverse the logic).

Reply:
got it. pushing once I finish testing locally.